dspace/config/spring/api/scripts.xml

```diff
@@ -97,4 +97,9 @@
         <property name="dspaceRunnableClass" value="org.dspace.app.bulkaccesscontrol.BulkAccessControlCli"/>
     </bean>
 
+    <bean id="import-oaire-suggestions" class="org.dspace.app.suggestion.oaire.PublicationLoaderCliScriptConfiguration">
+       <property name="description" value="Import and store locally metadata from OpenAIRE"/>
```

Comment:
I think this description is wrong?  Shouldn't it say something like "Import suggested publications from OpenAIRE"?

Reply:
What suggestions do, at the end of the line, is updating an existing repository item on its metadatas. External Objects are a different thing.